OpenRA.Mods.Common/Traits/Render/RenderSprites.cs

```diff
@@ -150,7 +150,15 @@ public static Func<WAngle> MakeFacingFunc(Actor self)
 			if (facing == null)
 				return () => WAngle.Zero;
 
-			return () => facing.Facing;
+			// Prevent jittering.
+			var cachedFacing = facing.Facing;
+			return () =>
+			{
+				if (Math.Abs((cachedFacing - facing.Facing).Facing) > 1)
```

Comment:
Since you're asking for performance improvements: Subtracting two `WAngle` structs creates a new struct. I suggest
```suggestion
				if (cachedFacing != facing.Facing && Math.Abs((cachedFacing - facing.Facing).Facing) > 1)
```
which also saves the `Math.Abs` call for most cases (facing does not change).

Reply:
Actually the `Math.Abs` call is not needed at all, the `WAngle` ctor already normalizes the facing.

Reply:
I went with the alternative. Also I've changed to comparing Angle as its a variable (not a func) and also doesn't need extra calculations.